test/unit/mdc-menu/simple.foundation.test.js

```diff
@@ -1,14 +1,14 @@
 /**
  * Copyright 2016 Google Inc. All Rights Reserved.
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
+ * Licensed under the Apache License, Version 2.0 (the 'License');
```

Comment:
I assume some kind of mass find/replace altered these comments in the copyright; we should revert these 2 edits

Reply:
great eye!